### DISCUSSION TITLE
Fuzzy Dates

### DISCUSSION THREAD

**Discussion by TedTschopp** [mmistakes/minimal-mistakes#3062] [Ideas]
I don't see any way to flag dates to be displayed as fuzzy.  For example: "just over 2 years ago during the night, on Wednesday, January 23rd"

I have done something like this on one of my Github Pages blog and I would be willing to start moving it over, I just expect my persona janky code isn't up to the standards and I am curious:

1. Would people want this?
2. Is there any coding standards or the like that should be followed if I just submit something.

Ted

**Comment by mmistakes** [comment]
Dates are hard with Jekyll. Even harder with this theme as I can’t think of how you’d handle translating/localizing those sorts of dates in multiple languages.

Pretty sure I’ve seen plugins that do this if you search around. I’ve done something similar with a Gatsby based blog. Though removed it because “time passed” dates aren’t as useful for a static blog.

For example you could publish a post now, it would say “published X amount of seconds ago”

Someone views it hours or days later and it still says that same inaccurate time. I guess if you’re publishing regularly this is less of a problem…

**Reply by TedTschopp** [reply]
Those are all good points.  The one I have elsewhere is a giant mess of Switch statements that drop you through the various permutations.  The internationalization would make that even worse.

**Comment by mmistakes** [comment]
Not something I’m likely to add to the theme, but you could customize it and use JavaScript to do it. That would get around the drawback I noted above with needing to publish regularly.

Using a small library like this https://timeago.org/
You could replace the publish dates the theme already exposes with a time ago, doing so on page load. Which would be way more accurate than having to publish all the time to get around static site limitations.
